spec/support/db_housekeeping.rb

```diff
@@ -20,7 +20,7 @@ def self.clean(seed: true)
       search_queries
     )
     tables.each do |table|
-      ActiveRecord::Base.connection.execute("TRUNCATE #{table}")
+      ActiveRecord::Base.connection.execute("TRUNCATE #{table} CASCADE")
```

Comment:
There is a DatabaseCleaner gem available...

Reply:
This minor change is good enough unless there's a compelling reason to use the gem?